backend-models/service-tests/profileservice-tests.txt.txt

```diff
@@ -0,0 +1,232 @@
+profileservice-tests.txt
+
+
+Before tests start, create 2 dummy accounts, one as teacher, another as student. Examples for names, 
```

Comment:
Do we need to create new accounts every time we test? Isn't creating a teacher dummy account the same as fulfilling this kind of pre-req?

{ (checklogin) IS TRUE && 
    (hasAccess($userid, $cid, 'w') || isSuperUser($userid) || hasAccess($userid, $cid, 'st')) } IS TRUE